### PR TITLE
Feature/ge 684

### PR DESCRIPTION
**Pull request by AltusBarry**

### CODE REVIEW

File: authentication_service/management/commands/demo_content.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import os
```
Comment (cobusc) [inline authentication_service/management/commands/demo_content.py]: Add comment that this management command is only intended for use when setting up a demo environment.

File: authentication_service/management/commands/demo_content.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+from django.contrib.auth.models import User
+from django.core.management.base import BaseCommand, CommandError
+
+from oidc_provider.models import Client
+
+
+class Command(BaseCommand):
```
Comment (cobusc) [inline authentication_service/management/commands/demo_content.py]: Add `help = 'Setup used for demonstration purposes only'` or something to that effect.

File: scripts/waitFor.sh [hunk 3]
```diff
@@ -1,25 +1,8 @@
...
-# Basic poller to check if the postgres service is available. Has dependency on
-# psql being available.
-
-# At this stage it uses the Docker docs example script:
-# https://docs.docker.com/compose/startup-order/
-#until psql -h "$DB_HOST" -U "postgres" -c '\q'; do
-#    >&2 echo "Postgres is unavailable - sleeping"
-#    sleep 1
-#done
+until nc -z ${DB_HOST:=db} ${DB_PORT:=5432} > /dev/null; do
```
Comment (cobusc) [inline scripts/waitFor.sh]: I cannot see where `db` is defined. Is there a docker compose service called `db` which it defaults to?

Comment (AltusBarry) [reply scripts/waitFor.sh]: Docker compose service name can be used as domain name. In this case, db is the db service setup in compose.

**Pull request merged**